packages/runtime/src/store/tutorial-runner.ts

```diff
@@ -1,5 +1,6 @@
 import type { CommandsSchema, Files } from '@tutorialkit/types';
 import type { IFSWatcher, WebContainer, WebContainerProcess } from '@webcontainer/api';
+import picomatch from 'picomatch';
```

Comment:
I tried `picomatch/posix` first, but the typescript intellisense complained

Reply:
Maybe Astro is polyfilling the node built-ins here. 🤔 
But I think it's better to use the `picomatch/posix` as we target browsers and webcontainers' file system.

The `@types/picomatch` are missing this entrypoint. You'll need to add something like this:
https://github.com/stackblitz/tutorialkit/blob/e288f4ed252a11f228a7dc27701bc91a5271c113/packages/react/src/types.d.ts#L1-L4

Reply:
I added the types so that `picomatch/posix` works, but then the e2e tests were failing, so I set the import to `picomatch/posix.js` with the corresponding types.  🤷🏻 I think it works?